project/npda/views/home.py

```diff
@@ -20,26 +22,29 @@
 # Logging
 logger = logging.getLogger(__name__)
 
+
+@sync_to_async
 @login_and_otp_required()
```

Comment:
This decorator isn't async compatible so you can fudge it by wrapping then unwrapping. It does break our login logging though (see the TODO MRB in decorators.py above). Django does [allow login_required to be applied to async views](https://code.djangoproject.com/ticket/31949) so it's probably just a case of changing our decorator to be written in the same way